### PR TITLE
feat: new homepage design

### PR DESCRIPTION
**Pull request by markshenouda**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

Screenshot 2024-07-03 at 1 58 46 PM

closes #ISSUE_NUMBER_GOES_HERE

...

## Description

...

## Changes

- High level
- changes that
- you made
- ...

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -36,4 +55,130 @@ import HeroCTA from "../features/landing/sections/HeroCTA.astro"
...
+    margin-top: var(--space-8x);
+  }
+
+  .recommended a {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: var(--space-4x) var(--space-5x);
+    box-shadow: 0px var(--space-5x) var(--space-8x) calc(-1 * var(--space-5x)) rgba(12, 22, 44, 0.24);
```
Comment (Zelig880) [inline src/pages/index.astro:131]: the shadow is "12px" not "20x"

`box-shadow: 0px 12px 32px -12px #0C162C1F;`

**Review by Zelig880 — CHANGES_REQUESTED** [review]
Good job! Just a few small points, but overall great PR

File: src/pages/index.astro [hunk 2]
```diff
@@ -2,28 +2,47 @@
...
+    <div class="heroContainer">
+      <div class="hero">
+        <h2>What are you <br />building?</h2>
+        <ProductTabs />
+      </div>
+    </div>
+    <div class="recommended">
+      <h4>Recommded reading</h4>
+      <h3>We think you'd love to explore</h3>
+      <h5>Popular searches</h5>
```
[the remark this answers is not in the document]
Comment (markshenouda) [reply src/pages/index.astro:23]: This is on the mobile design
Screenshot 2024-07-03 at 3 05 55 PM

File: src/pages/index.astro (hunk 1 above)
Comment (markshenouda) [reply src/pages/index.astro:131]: It didn't look the same as Figma so I tried to add few pixels but I'll revert it back

File: src/pages/index.astro [hunk 3]
```diff
@@ -2,28 +2,47 @@
...
 
-<LandingLayout>
-  <HeroCTA />
-  <div class="home-wrapper" id="article">
-    <ProductTabs />
-  </div>
-</LandingLayout>
+<BaseLayout title={formattedContentTitle}>
+  <main>
+    <div class="heroContainer">
```
[the remark this answers is not in the document]
Comment (Zelig880) [reply src/pages/index.astro:14]: This was replied in chat! :)

**Review by Zelig880 — APPROVED** [review]

**Pull request merged**